src/components/Tile/Tile.js

```diff
@@ -1,49 +1,117 @@
-import React, { useEffect, useRef } from 'react';
+import React, { useEffect, useMemo, useRef } from 'react';
 import './Tile.css';
 
+function getTrackUnavailableMessage(kind, trackState) {
+  if (!trackState) return;
+  switch (trackState.state) {
+    case 'blocked':
+      if (trackState.blocked.byPermissions) {
+        return `${kind} permission denied`;
+      } else if (trackState.blocked.byDeviceMissing) {
+        return `${kind} device missing`;
+      }
+    case 'off':
+      if (trackState.off.byUser) {
+        return `${kind} off`;
```

Comment:
Nitpick but I might add why it's off the the message. `Video turned off by user` or something so there's no wondering if it's a bug or not.

Reply:
hmm, fair point. what do you think of just "muted"?